src/map/utils/mobutils.h

```diff
@@ -66,8 +66,8 @@ namespace mobutils
 
     uint16 GetWeaponDamage(CMobEntity* PMob, uint16 slot);
     uint16 GetMagicEvasion(CMobEntity* PMob);
-    uint16 GetEvasion(CMobEntity* PMob);
```

Comment:
How is this possible? named in header as `GetEvasion` and in .cpp as GetDefense`?

Reply:
I was just talking to teo about this...I have no idea how that happened. GetEvasion was deleted so long ago.

Reply:
This proves all this really works via black magic.

Reply:
I don't want to talk about magic at 3am...